corehq/apps/cloudcare/views.py

```diff
@@ -626,20 +626,22 @@ class BlockWebAppsView(BaseDomainView):
     template_name = 'block_web_apps.html'
 
     def get(self, request, *args, **kwargs):
-        context = get_context_for_ucr_limit_error(request.domain)
+        mobile_ucr_count = get_mobile_ucr_count(request.domain)
+        context = self.get_context_for_ucr_limit_error(request.domain, mobile_ucr_count)
         return render(request, self.template_name, context)
 
-
-def get_context_for_ucr_limit_error(domain):
-    return {
-        'domain': domain,
-        'ucr_limit': settings.MAX_MOBILE_UCR_LIMIT,
-        'error_message': _("""You have the MOBILE_UCR feature flag enabled, and have exceeded the maximum limit
-                           of {ucr_limit} total User Configurable Reports used across all of your applications.
-                           To resolve, you must remove references to UCRs in your applications until you are under
-                           the limit. If you believe this is a mistake, please reach out to support.
-                           """).format(ucr_limit=settings.MAX_MOBILE_UCR_LIMIT)
-    }
+    @classmethod
+    def get_context_for_ucr_limit_error(cls, domain, mobile_ucr_count):
```

Comment:
nit: Looks like this could be a `staticmethod` since `cls` is not referenced.

Reply:
See b959e0f067a2899edb971ad972cece6e5fcf6ade